src/Tribe/Metabox.php

```diff
@@ -473,6 +471,36 @@ public function has_permission( $post, $data, $nonce_action ) {
 		return ! empty( $data['nonce'] ) && wp_verify_nonce( $data['nonce'], $nonce_action ) && current_user_can( get_post_type_object( $post->post_type )->cap->edit_posts );
 	}
 
+	/**
+	 * Tests if the user has the specified capability in relation to whatever post type
+	 * the attendee object relates to.
+	 *
+	 * For example, if the attendee was generated for a ticket set up in relation to a
+	 * post of the banana type, the generic capability "edit_posts" will be mapped to
```

Comment:
What about quoting `banana` with backtick here to distinguish it from normal words?